src/validation/emailValidation.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+import { regexForEmail } from '../__mocks__/emailVerification';
```

Comment:
abstracting `regexForEmail` is a good idea but I think you shouldn't put it in the `__mocks__` folder. you can create a `./src/helpers/defaults` file and include things like this.

Reply:
Alright, it has been done!